src/main/resources/com/google/api/codegen/go/discovery_fragment.snip

```diff
@@ -113,35 +132,74 @@ func main() {
 @end
 
 @private initialize(method)
-  @join param : context.getFlatMethodParams(method) on BREAK.add(BREAK)
-    @let signatureType = context.getApiaryConfig.getType(method.getRequestTypeUrl), \
-        paramField = context.getField(signatureType, param), \
-        paramValue = context.typeDefaultValue(signatureType, paramField), \
-        paramDescription = context.getApiaryConfig.getDescription(signatureType.getName, param)
-      {@description(paramDescription)}
-      {@unclashVar(param)} := {@paramValue} // {@TODO()} Update placeholder value.
+  @let methodName = context.lowerCamelToUpperCamel(context.getMethodName(method)), \
+       signatureType = context.getApiaryConfig.getType(method.getRequestTypeUrl), \
+       params = context.getFlatMethodParams(method)
+    @if params
+      @join param : params
+        @let paramField = context.getField(signatureType, param), \
+             paramValue = context.typeDefaultValue(signatureType, paramField), \
+             paramDescription = context.getDescription(signatureType.getName, param)
+          {@description(paramDescription)}
+          {@unclashVar(param)} := {@paramValue} // {@TODO()} Update placeholder value(s).
```

Comment:
Seems to me we should be able to know whether there is one or more placeholders, so we could use "value" or "values" rather than "value(s)". This ties in with the work @pongad is doing in #249 .

Maybe one of you could do the Java plumbing so that we can eventually get a paramNumPlaceholders and properly pluralize. I'm fine if we do this in a chaser PR, but if so, please file an issue so we don't forget.